### PR TITLE
Fix a java top 10 crasher reported on sentry

### PR DESCRIPTION
**Pull request by nirvn**
Fixes https://sentry.io/organizations/opengisch/issues/3409948078/?project=6119013&query=is%3Aunresolved+release%3A%22ch.opengis.qfield%402.2.1+-+Coordinated+Capybara%2B2020199%2A%22&sort=freq&statsPeriod=7d

### CODE REVIEW

**Comment by qfield-fairy** [comment]
🎉 Ta-daaa, freshly created APKs are available for 64c2ba092687653a1b2a6a7f2589696214c2200b: [**arm64-android**](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-3143-ba580b8cc3348621a002bb1737b76fe6a1fd2cbd-arm64-android.apk)
<details>
<summary>Other architectures</summary>

- [arm-android](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-3143-ba580b8cc3348621a002bb1737b76fe6a1fd2cbd-arm-android.apk)
- [x64-android](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-3143-ba580b8cc3348621a002bb1737b76fe6a1fd2cbd-x64-android.apk)
- [x86-android](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-3143-ba580b8cc3348621a002bb1737b76fe6a1fd2cbd-x86-android.apk)
</details>

File: platform/android/src/ch/opengis/qfield/QFieldActivity.java [hunk 1]
```diff
@@ -615,15 +617,23 @@ public void run() {
...
-                    AlertDialog alertDialog =
-                        new AlertDialog
-                            .Builder(QFieldActivity.this, R.style.DialogTheme)
-                            .create();
-                    alertDialog.setTitle(getString(R.string.import_error));
-                    alertDialog.setMessage(
-                        getString(R.string.import_dataset_error));
                     if (!isFinishing()) {
-                        alertDialog.show();
+                        new Handler(Looper.getMainLooper())
```
Comment (m-kuhn) [inline platform/android/src/ch/opengis/qfield/QFieldActivity.java:621]: Could this code be factored out into a separate method (or Runnable class)?

**Comment by nirvn** [comment]
@m-kuhn , done.

**Review by m-kuhn — APPROVED** [review]
Excellent, thanks!

**Pull request merged**